Advanced-Solidity.md

```diff
@@ -130,12 +130,12 @@ contract Enum {
         return status;
     }
 
-    // Pass a uint for input to update the value
+    // Pass a Status enum value to update the status
```

Comment:
This is true, but I think the previous comment was written in the context of using this function from the frontend.

> Since they are just an abstraction for human readable constants, in actuality, they are internally represented as `unit`s.

Of course, you can create an enum from the frontend as well, but I think maybe we can modify it to suit both intentions. 

Maybe you can say - 
`// Pass the desired Status enum value as a uint`